2025-07-19 12:18:23,576 - INFO - Updating order 5
2025-07-19 12:18:23,576 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 12:18:24,357 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 500 Internal Server Error"
2025-07-19 12:18:26,360 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 12:18:27,151 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 500 Internal Server Error"
2025-07-19 12:18:29,159 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 12:18:29,926 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 500 Internal Server Error"
2025-07-19 12:24:37,365 - INFO - Updating order 5
2025-07-19 12:24:37,475 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 12:24:38,372 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 200 OK"
2025-07-19 12:24:38,376 - INFO - Making GET request to http://localhost:8001/items/3
2025-07-19 12:24:39,113 - INFO - HTTP Request: GET http://localhost:8001/items/3 "HTTP/1.1 200 OK"
2025-07-19 12:24:39,303 - INFO - select pg_catalog.version()
2025-07-19 12:24:39,304 - INFO - [raw sql] ()
2025-07-19 12:24:39,308 - INFO - select current_schema()
2025-07-19 12:24:39,318 - INFO - [raw sql] ()
2025-07-19 12:24:39,323 - INFO - show standard_conforming_strings
2025-07-19 12:24:39,324 - INFO - [raw sql] ()
2025-07-19 12:24:39,329 - INFO - BEGIN (implicit)
2025-07-19 12:24:39,341 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 12:24:39,367 - INFO - [generated in 0.02587s] (5,)
2025-07-19 12:24:39,383 - INFO - UPDATE orders SET quantity=$1::INTEGER, total_price=$2::FLOAT WHERE orders.id = $3::INTEGER
2025-07-19 12:24:39,386 - INFO - [generated in 0.00445s] (3, 15.0, 5)
2025-07-19 12:24:39,408 - INFO - COMMIT
2025-07-19 12:24:39,414 - INFO - BEGIN (implicit)
2025-07-19 12:24:39,419 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 12:24:39,420 - INFO - [generated in 0.00195s] (5,)
2025-07-19 12:24:39,424 - INFO - ROLLBACK
2025-07-19 12:24:58,291 - INFO - Deleting order 5
2025-07-19 12:24:58,293 - INFO - BEGIN (implicit)
2025-07-19 12:24:58,296 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 12:24:58,303 - INFO - [cached since 18.96s ago] (5,)
2025-07-19 12:24:58,311 - INFO - DELETE FROM orders WHERE orders.id = $1::INTEGER
2025-07-19 12:24:58,315 - INFO - [generated in 0.00383s] (5,)
2025-07-19 12:24:58,321 - INFO - COMMIT
2025-07-19 12:25:01,919 - INFO - Fetching order 5
2025-07-19 12:25:01,919 - INFO - BEGIN (implicit)
2025-07-19 12:25:01,919 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 12:25:01,919 - INFO - [cached since 22.58s ago] (5,)
2025-07-19 12:25:01,927 - INFO - ROLLBACK
2025-07-19 12:25:05,612 - INFO - Fetching all orders
2025-07-19 12:25:05,613 - INFO - BEGIN (implicit)
2025-07-19 12:25:05,615 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders
2025-07-19 12:25:05,618 - INFO - [generated in 0.00265s] ()
2025-07-19 12:25:05,623 - INFO - ROLLBACK
2025-07-19 12:26:37,759 - INFO - Fetching all orders
2025-07-19 12:26:37,760 - INFO - BEGIN (implicit)
2025-07-19 12:26:37,764 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders
2025-07-19 12:26:37,766 - INFO - [cached since 92.15s ago] ()
2025-07-19 12:26:37,771 - INFO - ROLLBACK
2025-07-19 12:41:20,808 - INFO - Fetching all orders
2025-07-19 12:41:20,809 - INFO - BEGIN (implicit)
2025-07-19 12:41:20,812 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders
2025-07-19 12:41:20,814 - INFO - [cached since 975.2s ago] ()
2025-07-19 12:41:20,818 - INFO - ROLLBACK
2025-07-19 12:42:28,065 - INFO - Creating order
2025-07-19 12:42:28,073 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 12:42:29,289 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 12:42:29,293 - INFO - BEGIN (implicit)
2025-07-19 12:42:29,300 - INFO - INSERT INTO orders (user_id, item_id, quantity, total_price, status) VALUES ($1::INTEGER, $2::INTEGER, $3::INTEGER, $4::FLOAT, $5::VARCHAR) RETURNING orders.id
2025-07-19 12:42:29,304 - INFO - [generated in 0.00493s] (3, 2, 5, 250.0, 'pending')
2025-07-19 12:42:29,309 - INFO - COMMIT
2025-07-19 12:42:29,311 - INFO - BEGIN (implicit)
2025-07-19 12:42:29,312 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 12:42:29,313 - INFO - [cached since 1070s ago] (6,)
2025-07-19 12:42:29,316 - INFO - ROLLBACK
2025-07-19 12:42:34,077 - INFO - Fetching all orders
2025-07-19 12:42:34,085 - INFO - BEGIN (implicit)
2025-07-19 12:42:34,085 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders
2025-07-19 12:42:34,093 - INFO - [cached since 1048s ago] ()
2025-07-19 12:42:34,093 - INFO - ROLLBACK
2025-07-19 12:48:38,383 - INFO - Fetching all orders
2025-07-19 12:48:38,396 - ERROR - Unhandled exception: 'dict' object has no attribute 'user_id'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 28, in get_all
    return await get_all_orders(db, user.user_id)
                                    ^^^^^^^^^^^^
AttributeError: 'dict' object has no attribute 'user_id'
2025-07-19 12:49:29,274 - INFO - Fetching all orders
2025-07-19 12:49:29,413 - INFO - select pg_catalog.version()
2025-07-19 12:49:29,413 - INFO - [raw sql] ()
2025-07-19 12:49:29,416 - INFO - select current_schema()
2025-07-19 12:49:29,416 - INFO - [raw sql] ()
2025-07-19 12:49:29,418 - INFO - show standard_conforming_strings
2025-07-19 12:49:29,418 - INFO - [raw sql] ()
2025-07-19 12:49:29,421 - INFO - BEGIN (implicit)
2025-07-19 12:49:29,427 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 12:49:29,428 - INFO - [generated in 0.00135s] (3,)
2025-07-19 12:49:29,435 - INFO - ROLLBACK
2025-07-19 12:50:20,383 - INFO - Fetching all orders
2025-07-19 12:50:20,384 - INFO - BEGIN (implicit)
2025-07-19 12:50:20,386 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 12:50:20,388 - INFO - [cached since 50.96s ago] (2,)
2025-07-19 12:50:20,391 - INFO - ROLLBACK
2025-07-19 12:50:29,987 - INFO - Fetching all orders
2025-07-19 12:50:29,988 - INFO - BEGIN (implicit)
2025-07-19 12:50:29,989 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 12:50:29,991 - INFO - [cached since 60.56s ago] (3,)
2025-07-19 12:50:29,993 - INFO - ROLLBACK
2025-07-19 13:05:27,428 - INFO - Fetching all orders
2025-07-19 13:05:27,429 - INFO - BEGIN (implicit)
2025-07-19 13:05:27,430 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 13:05:27,431 - INFO - [cached since 958s ago] (2,)
2025-07-19 13:05:27,433 - INFO - ROLLBACK
2025-07-19 13:05:39,542 - INFO - Fetching all orders
2025-07-19 13:05:39,543 - INFO - BEGIN (implicit)
2025-07-19 13:05:39,544 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 13:05:39,545 - INFO - [cached since 970.1s ago] (2,)
2025-07-19 13:05:39,547 - INFO - ROLLBACK
2025-07-19 13:06:01,262 - INFO - Creating order
2025-07-19 13:06:01,263 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:02,053 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 13:06:04,049 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:04,608 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 13:06:06,611 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:07,195 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 13:06:23,811 - INFO - Creating order
2025-07-19 13:06:23,811 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:24,403 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 13:06:26,420 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:27,022 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 13:06:29,016 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 13:06:29,577 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 17:45:30,560 - INFO - Fetching all orders
2025-07-19 17:45:30,990 - INFO - select pg_catalog.version()
2025-07-19 17:45:30,990 - INFO - [raw sql] ()
2025-07-19 17:45:30,993 - INFO - select current_schema()
2025-07-19 17:45:30,993 - INFO - [raw sql] ()
2025-07-19 17:45:30,995 - INFO - show standard_conforming_strings
2025-07-19 17:45:30,995 - INFO - [raw sql] ()
2025-07-19 17:45:31,000 - INFO - BEGIN (implicit)
2025-07-19 17:45:31,008 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 17:45:31,009 - INFO - [generated in 0.00113s] (2,)
2025-07-19 17:45:31,017 - INFO - ROLLBACK
2025-07-19 17:46:24,525 - INFO - BEGIN (implicit)
2025-07-19 17:46:24,528 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 17:46:24,530 - INFO - [generated in 0.00234s] (1,)
2025-07-19 17:46:24,534 - INFO - Deleting order 1
2025-07-19 17:46:24,539 - INFO - DELETE FROM orders WHERE orders.id = $1::INTEGER
2025-07-19 17:46:24,540 - INFO - [generated in 0.00086s] (1,)
2025-07-19 17:46:24,556 - INFO - COMMIT
2025-07-19 17:46:24,561 - INFO - Making PUT request to http://localhost:8001/items/1/increase
2025-07-19 17:46:25,550 - INFO - HTTP Request: PUT http://localhost:8001/items/1/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:46:25,552 - ERROR - Failed to increase inventory for item 1
2025-07-19 17:46:27,552 - INFO - Making PUT request to http://localhost:8001/items/1/increase
2025-07-19 17:46:28,604 - INFO - HTTP Request: PUT http://localhost:8001/items/1/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:46:28,605 - ERROR - Failed to increase inventory for item 1
2025-07-19 17:46:30,620 - INFO - Making PUT request to http://localhost:8001/items/1/increase
2025-07-19 17:46:31,684 - INFO - HTTP Request: PUT http://localhost:8001/items/1/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:46:31,747 - ERROR - Failed to increase inventory for item 1
2025-07-19 17:46:53,572 - INFO - BEGIN (implicit)
2025-07-19 17:46:53,573 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 17:46:53,575 - INFO - [cached since 29.05s ago] (2,)
2025-07-19 17:46:53,577 - INFO - Deleting order 2
2025-07-19 17:46:53,579 - INFO - DELETE FROM orders WHERE orders.id = $1::INTEGER
2025-07-19 17:46:53,582 - INFO - [cached since 29.04s ago] (2,)
2025-07-19 17:46:53,591 - INFO - COMMIT
2025-07-19 17:46:53,595 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:46:54,320 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:46:54,494 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:46:56,507 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:46:57,577 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:46:57,580 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:46:59,591 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:47:00,415 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:47:00,417 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:48:33,142 - INFO - Fetching all orders
2025-07-19 17:48:33,142 - INFO - BEGIN (implicit)
2025-07-19 17:48:33,143 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 17:48:33,151 - INFO - [cached since 182.1s ago] (2,)
2025-07-19 17:48:33,155 - INFO - ROLLBACK
2025-07-19 17:49:05,046 - INFO - Creating order
2025-07-19 17:49:05,050 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:49:05,654 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:49:05,655 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:49:06,289 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:49:06,290 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:49:06,915 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:49:06,916 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:49:08,924 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:49:09,553 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:49:09,557 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:49:11,580 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:49:12,313 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 500 Internal Server Error"
2025-07-19 17:49:12,320 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:55:38,040 - INFO - Creating order
2025-07-19 17:55:38,041 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:55:39,729 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:55:39,731 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:55:40,472 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:55:40,475 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:55:41,292 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:55:41,293 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:55:43,308 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:55:43,887 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:55:43,888 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:55:45,890 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:55:46,516 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:55:46,611 - ERROR - Failed to reduce inventory for item 1
2025-07-19 17:56:50,449 - INFO - BEGIN (implicit)
2025-07-19 17:56:50,450 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 17:56:50,450 - INFO - [cached since 625.9s ago] (3,)
2025-07-19 17:56:50,452 - INFO - Deleting order 3
2025-07-19 17:56:50,453 - INFO - DELETE FROM orders WHERE orders.id = $1::INTEGER
2025-07-19 17:56:50,454 - INFO - [cached since 625.9s ago] (3,)
2025-07-19 17:56:50,459 - INFO - COMMIT
2025-07-19 17:56:50,467 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:56:51,105 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:56:51,107 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:56:53,130 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:56:53,713 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:56:53,716 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:56:55,719 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 17:56:56,348 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 422 Unprocessable Entity"
2025-07-19 17:56:56,349 - ERROR - Failed to increase inventory for item 2
2025-07-19 17:58:58,414 - INFO - select pg_catalog.version()
2025-07-19 17:58:58,415 - INFO - [raw sql] ()
2025-07-19 17:58:58,418 - INFO - select current_schema()
2025-07-19 17:58:58,418 - INFO - [raw sql] ()
2025-07-19 17:58:58,420 - INFO - show standard_conforming_strings
2025-07-19 17:58:58,421 - INFO - [raw sql] ()
2025-07-19 17:58:58,423 - INFO - BEGIN (implicit)
2025-07-19 17:58:58,430 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 17:58:58,435 - INFO - [generated in 0.00568s] (3,)
2025-07-19 17:58:58,442 - INFO - ROLLBACK
2025-07-19 17:59:06,677 - INFO - Creating order
2025-07-19 17:59:06,678 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:59:07,509 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:59:07,512 - INFO - Making GET request to http://localhost:8001/items/1
2025-07-19 17:59:08,157 - INFO - HTTP Request: GET http://localhost:8001/items/1 "HTTP/1.1 200 OK"
2025-07-19 17:59:08,159 - INFO - Making PUT request to http://localhost:8001/items/1/decrease
2025-07-19 17:59:08,823 - INFO - HTTP Request: PUT http://localhost:8001/items/1/decrease "HTTP/1.1 200 OK"
2025-07-19 17:59:08,825 - INFO - BEGIN (implicit)
2025-07-19 17:59:08,829 - INFO - INSERT INTO orders (user_id, item_id, quantity, total_price, status) VALUES ($1::INTEGER, $2::INTEGER, $3::INTEGER, $4::FLOAT, $5::VARCHAR) RETURNING orders.id
2025-07-19 17:59:08,833 - INFO - [generated in 0.00430s] (2, 1, 10, 50.0, 'pending')
2025-07-19 17:59:08,838 - INFO - COMMIT
2025-07-19 17:59:08,842 - INFO - BEGIN (implicit)
2025-07-19 17:59:08,852 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 17:59:08,855 - INFO - [generated in 0.00299s] (7,)
2025-07-19 17:59:08,859 - INFO - ROLLBACK
2025-07-19 18:00:57,490 - INFO - Fetching all orders
2025-07-19 18:00:57,491 - INFO - BEGIN (implicit)
2025-07-19 18:00:57,492 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:00:57,493 - INFO - [generated in 0.00106s] (3,)
2025-07-19 18:00:57,496 - INFO - ROLLBACK
2025-07-19 18:01:19,668 - INFO - Creating order
2025-07-19 18:01:19,669 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 18:01:20,290 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 18:01:22,301 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 18:01:22,910 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 18:01:24,919 - INFO - Making GET request to http://localhost:8001/items/5
2025-07-19 18:01:25,655 - INFO - HTTP Request: GET http://localhost:8001/items/5 "HTTP/1.1 404 Not Found"
2025-07-19 18:01:52,658 - INFO - Creating order
2025-07-19 18:01:52,658 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 18:01:53,308 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 18:01:53,309 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 18:01:53,936 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 18:01:53,937 - INFO - Making PUT request to http://localhost:8001/items/2/decrease
2025-07-19 18:01:54,542 - INFO - HTTP Request: PUT http://localhost:8001/items/2/decrease "HTTP/1.1 200 OK"
2025-07-19 18:01:54,544 - INFO - BEGIN (implicit)
2025-07-19 18:01:54,549 - INFO - INSERT INTO orders (user_id, item_id, quantity, total_price, status) VALUES ($1::INTEGER, $2::INTEGER, $3::INTEGER, $4::FLOAT, $5::VARCHAR) RETURNING orders.id
2025-07-19 18:01:54,550 - INFO - [cached since 165.7s ago] (3, 2, 5, 250.0, 'pending')
2025-07-19 18:01:54,551 - INFO - COMMIT
2025-07-19 18:01:54,553 - INFO - BEGIN (implicit)
2025-07-19 18:01:54,553 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:01:54,556 - INFO - [cached since 165.7s ago] (8,)
2025-07-19 18:01:54,557 - INFO - ROLLBACK
2025-07-19 18:02:39,700 - INFO - Fetching all orders
2025-07-19 18:02:39,701 - INFO - BEGIN (implicit)
2025-07-19 18:02:39,702 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:02:39,703 - INFO - [cached since 102.2s ago] (3,)
2025-07-19 18:02:39,704 - INFO - ROLLBACK
2025-07-19 18:02:48,144 - INFO - Updating order 6
2025-07-19 18:02:48,144 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 18:02:48,876 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 200 OK"
2025-07-19 18:02:48,878 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 18:02:49,481 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 18:02:49,483 - INFO - BEGIN (implicit)
2025-07-19 18:02:49,484 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:02:49,485 - INFO - [cached since 231.1s ago] (6,)
2025-07-19 18:02:49,491 - INFO - UPDATE orders SET user_id=$1::INTEGER, quantity=$2::INTEGER, total_price=$3::FLOAT WHERE orders.id = $4::INTEGER
2025-07-19 18:02:49,493 - INFO - [generated in 0.00260s] (2, 50, 2500.0, 6)
2025-07-19 18:02:49,498 - INFO - COMMIT
2025-07-19 18:02:49,500 - INFO - BEGIN (implicit)
2025-07-19 18:02:49,501 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:02:49,501 - INFO - [cached since 220.6s ago] (6,)
2025-07-19 18:02:49,503 - INFO - ROLLBACK
2025-07-19 18:16:31,391 - INFO - Fetching all orders
2025-07-19 18:16:31,651 - INFO - select pg_catalog.version()
2025-07-19 18:16:31,652 - INFO - [raw sql] ()
2025-07-19 18:16:31,656 - INFO - select current_schema()
2025-07-19 18:16:31,657 - INFO - [raw sql] ()
2025-07-19 18:16:31,661 - INFO - show standard_conforming_strings
2025-07-19 18:16:31,662 - INFO - [raw sql] ()
2025-07-19 18:16:31,664 - INFO - BEGIN (implicit)
2025-07-19 18:16:31,677 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:16:31,679 - INFO - [generated in 0.00262s] (2,)
2025-07-19 18:16:31,700 - INFO - ROLLBACK
2025-07-19 18:16:48,842 - INFO - Fetching all orders
2025-07-19 18:16:48,843 - INFO - BEGIN (implicit)
2025-07-19 18:16:48,844 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:16:48,847 - INFO - [cached since 17.17s ago] (2,)
2025-07-19 18:16:48,850 - INFO - ROLLBACK
2025-07-19 18:36:32,994 - INFO - Fetching all orders
2025-07-19 18:36:33,190 - INFO - select pg_catalog.version()
2025-07-19 18:36:33,190 - INFO - [raw sql] ()
2025-07-19 18:36:33,194 - INFO - select current_schema()
2025-07-19 18:36:33,196 - INFO - [raw sql] ()
2025-07-19 18:36:33,199 - INFO - show standard_conforming_strings
2025-07-19 18:36:33,200 - INFO - [raw sql] ()
2025-07-19 18:36:33,202 - INFO - BEGIN (implicit)
2025-07-19 18:36:33,213 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:36:33,215 - INFO - [generated in 0.00290s] (2,)
2025-07-19 18:36:33,224 - INFO - ROLLBACK
2025-07-19 18:37:41,916 - INFO - Updating order 6
2025-07-19 18:37:41,916 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 18:37:43,378 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 200 OK"
2025-07-19 18:37:43,379 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 18:37:44,007 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 18:37:44,010 - INFO - BEGIN (implicit)
2025-07-19 18:37:44,012 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:37:44,013 - INFO - [generated in 0.00094s] (6,)
2025-07-19 18:37:44,018 - INFO - COMMIT
2025-07-19 18:37:44,024 - INFO - BEGIN (implicit)
2025-07-19 18:37:44,026 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:37:44,030 - INFO - [generated in 0.00418s] (6,)
2025-07-19 18:37:44,038 - INFO - ROLLBACK
2025-07-19 18:38:08,896 - INFO - Updating order 6
2025-07-19 18:38:08,897 - INFO - Making GET request to http://localhost:8000/users/2
2025-07-19 18:38:09,525 - INFO - HTTP Request: GET http://localhost:8000/users/2 "HTTP/1.1 200 OK"
2025-07-19 18:38:09,527 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 18:38:10,097 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
2025-07-19 18:38:10,132 - INFO - BEGIN (implicit)
2025-07-19 18:38:10,204 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:38:10,247 - INFO - [cached since 26.23s ago] (6,)
2025-07-19 18:38:10,301 - INFO - Making PUT request to http://localhost:8001/items/2/increase
2025-07-19 18:38:11,195 - INFO - HTTP Request: PUT http://localhost:8001/items/2/increase "HTTP/1.1 200 OK"
2025-07-19 18:38:11,199 - INFO - UPDATE orders SET quantity=$1::INTEGER, total_price=$2::FLOAT WHERE orders.id = $3::INTEGER
2025-07-19 18:38:11,203 - INFO - [generated in 0.00476s] (10, 250.0, 6)
2025-07-19 18:38:11,209 - INFO - COMMIT
2025-07-19 18:38:11,213 - INFO - BEGIN (implicit)
2025-07-19 18:38:11,214 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 18:38:11,215 - INFO - [cached since 27.19s ago] (6,)
2025-07-19 18:38:11,221 - INFO - ROLLBACK
2025-07-19 18:39:10,806 - INFO - Fetching all orders
2025-07-19 18:39:10,807 - INFO - BEGIN (implicit)
2025-07-19 18:39:10,808 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders 
WHERE orders.user_id = $1::INTEGER
2025-07-19 18:39:10,813 - INFO - [cached since 157.6s ago] (1,)
2025-07-19 18:39:10,928 - INFO - ROLLBACK
2025-07-19 19:10:58,602 - INFO - Fetching all orders
2025-07-19 19:10:58,684 - ERROR - Unhandled exception: 'role'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 32, in get_all
    return await get_all_orders(db, user["user_id"], user["role"])
                                                     ~~~~^^^^^^^^
KeyError: 'role'
2025-07-19 19:11:02,250 - INFO - Fetching all orders
2025-07-19 19:11:02,251 - ERROR - Unhandled exception: 'role'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 32, in get_all
    return await get_all_orders(db, user["user_id"], user["role"])
                                                     ~~~~^^^^^^^^
KeyError: 'role'
2025-07-19 19:16:41,129 - INFO - Fetching all orders
2025-07-19 19:16:41,461 - ERROR - Unhandled exception: 'role'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 32, in get_all
    return await get_all_orders(db, user["user_id"], user["role"])
                                                     ~~~~^^^^^^^^
KeyError: 'role'
2025-07-19 19:19:19,488 - INFO - Fetching all orders
2025-07-19 19:19:19,489 - ERROR - Unhandled exception: 'role'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 32, in get_all
    return await get_all_orders(db, user["user_id"], user["role"])
                                                     ~~~~^^^^^^^^
KeyError: 'role'
2025-07-19 19:21:43,850 - INFO - Fetching all orders
2025-07-19 19:21:43,851 - ERROR - Unhandled exception: 'role'
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 32, in get_all
    return await get_all_orders(db, user["user_id"], user["role"])
                                                     ~~~~^^^^^^^^
KeyError: 'role'
2025-07-19 19:25:18,660 - INFO - Fetching all orders
2025-07-19 19:25:18,890 - INFO - select pg_catalog.version()
2025-07-19 19:25:18,890 - INFO - [raw sql] ()
2025-07-19 19:25:18,901 - INFO - select current_schema()
2025-07-19 19:25:18,903 - INFO - [raw sql] ()
2025-07-19 19:25:18,912 - INFO - show standard_conforming_strings
2025-07-19 19:25:18,913 - INFO - [raw sql] ()
2025-07-19 19:25:18,914 - INFO - BEGIN (implicit)
2025-07-19 19:25:18,923 - INFO - SELECT orders.id, orders.user_id, orders.item_id, orders.quantity, orders.total_price, orders.status 
FROM orders
2025-07-19 19:25:18,925 - INFO - [generated in 0.00187s] ()
2025-07-19 19:25:18,933 - INFO - ROLLBACK
2025-07-19 19:25:51,384 - INFO - Fetching order 1
2025-07-19 19:25:51,393 - ERROR - Unhandled exception: cannot access local variable 'order' where it is not associated with a value
Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 151, in call_next
    message = await recv_stream.receive()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\anyio\streams\memory.py", line 126, in receive
    raise EndOfStream from None
anyio.EndOfStream

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\main.py", line 21, in prometheus_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 159, in call_next
    raise app_exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\base.py", line 144, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\middleware\exceptions.py", line 63, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 716, in __call__
    await self.middleware_stack(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 736, in app
    await route.handle(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 290, in handle
    await self.app(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 78, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 53, in wrapped_app
    raise exc
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\_exception_handler.py", line 42, in wrapped_app
    await app(scope, receive, sender)
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\starlette\routing.py", line 75, in app
    response = await f(request)
               ^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 302, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\clothenv\Lib\site-packages\fastapi\routing.py", line 213, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "F:\BROTOTYPE WEEK TASKS\WEEK 39\Project 31 - Clothing Store with Microservices\order_service\app\routers\order.py", line 37, in get
    order = await get_order_by_id(db, order_id, order["user_id"], user["role"])
                                                ^^^^^
UnboundLocalError: cannot access local variable 'order' where it is not associated with a value
2025-07-19 19:27:23,291 - INFO - Fetching order 1
2025-07-19 19:27:23,515 - INFO - select pg_catalog.version()
2025-07-19 19:27:23,515 - INFO - [raw sql] ()
2025-07-19 19:27:23,518 - INFO - select current_schema()
2025-07-19 19:27:23,518 - INFO - [raw sql] ()
2025-07-19 19:27:23,521 - INFO - show standard_conforming_strings
2025-07-19 19:27:23,530 - INFO - [raw sql] ()
2025-07-19 19:27:23,534 - INFO - BEGIN (implicit)
2025-07-19 19:27:23,565 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 19:27:23,570 - INFO - [generated in 0.00556s] (1,)
2025-07-19 19:27:23,591 - INFO - ROLLBACK
2025-07-19 19:27:28,882 - INFO - Fetching order 4
2025-07-19 19:27:28,883 - INFO - BEGIN (implicit)
2025-07-19 19:27:28,884 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 19:27:28,884 - INFO - [cached since 5.32s ago] (4,)
2025-07-19 19:27:28,886 - INFO - ROLLBACK
2025-07-19 19:27:32,793 - INFO - Fetching order 6
2025-07-19 19:27:32,794 - INFO - BEGIN (implicit)
2025-07-19 19:27:32,795 - INFO - SELECT orders.id AS orders_id, orders.user_id AS orders_user_id, orders.item_id AS orders_item_id, orders.quantity AS orders_quantity, orders.total_price AS orders_total_price, orders.status AS orders_status 
FROM orders 
WHERE orders.id = $1::INTEGER
2025-07-19 19:27:32,796 - INFO - [cached since 9.231s ago] (6,)
2025-07-19 19:27:32,797 - INFO - ROLLBACK
2025-07-19 20:16:23,616 - INFO - Creating order
2025-07-19 20:16:23,619 - INFO - Making GET request to http://localhost:8001/items/2
2025-07-19 20:16:24,889 - INFO - HTTP Request: GET http://localhost:8001/items/2 "HTTP/1.1 200 OK"
